tutorials/hello-kubernetes/README.md

```diff
@@ -20,11 +20,116 @@ cd quickstarts/tutorials/hello-kubernetes
 
 > **Note**: See https://github.com/dapr/quickstarts#supported-dapr-runtime-version for supported tags. Use `git clone https://github.com/dapr/quickstarts.git` when using the edge version of dapr runtime.
 
-## Step 1 - Setup Dapr on your Kubernetes cluster
-
 The first thing you need is an RBAC enabled Kubernetes cluster. This could be running on your machine using Minikube, or it could be a fully-fledged cluster in Azure using [AKS](https://azure.microsoft.com/en-us/services/kubernetes-service/).
 
-Once you have a cluster, follow the steps below to deploy Dapr to it. For more details, see [Deploy Dapr on a Kubernetes cluster](https://docs.dapr.io/operations/hosting/kubernetes/kubernetes-deploy/).
+## Using Dapr Mutli-app run
```

Comment:
I would add "Using Multi-app run" and "Using the kubectl CLI" into separate tabs to make this the whole tutorial shorter.

Reply:
Could you explain this again?

Reply:
I dont think we can do that in GH